spacy_llm/backends/rest/backend/openai.py

```diff
@@ -14,7 +14,7 @@ class Endpoints(str, Enum):
     NON_CHAT = "https://api.openai.com/v1/completions"
 
 
-class OpenAIBackend(Backend):
+class OpenAIRESTBackend(Backend):
```

Comment:
nit but I think we need a rename of the base rest backend as well right?

To like `RESTBackend` or something. `REST` is kind of a weird name though btw. `HttpBackend` and `OpenAIHttpBackend` might make more sense?

Reply:
Eh, that was a refactoring slip-up. I renamed it back. I'm open to renaming REST -> HTTP though.